tesseract-server/src/handlers/aggregate.rs

```diff
@@ -97,47 +98,46 @@ pub fn do_aggregate(
     // this is in braces to explicitly the scope in which
     // req is borrowed, since req is moved later in the `map_err`
     {
-        let cache = req.state().cache.read().unwrap();
+        let cache = state.cache.read().unwrap();
         let cube_cache = some_or_404!(cache.find_cube_info(&cube), format!("Cube {} not found", cube));
```

Comment:
Noting that this validation caused complete failure of request when I wanted to run without the cache (it was taking a very long time). Is this the desired behavior?